tests/test_wave_01.py

```diff
@@ -158,13 +158,14 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
+    assert MOVIE_TITLE_1 in updated_data["watched"][0].values()
```

Comment:
Rather than checking against _all_ values, we'd still want to compare against the value stored under the `"title"` key
```py
    assert MOVIE_TITLE_1 == updated_data["watched"][0]["title"]
```

But really, we'd want to ensure that all the data is there as expected. To do so , we can make our own dictionary to compare with
```py
    assert updated_data["watched"][0] == {
        "title": MOVIE_TITLE_1,
        "genre": GENRE_1,
        "rating": RATING_1
    }
```

And though we know there's only a single movie in the list this time, since the specification doesn't require that movies be moved to a particular location in the watched list, we could use an `in` check so the location doesn't matter.
```py
    expected_movie = {
        "title": MOVIE_TITLE_1,
        "genre": GENRE_1,
        "rating": RATING_1
    }
    assert expected_movie in updated_data["watched"]
```